src/main/lombok/org/heigit/ohsome/ohsomeapi/inputprocessing/ResourceParameters.java

```diff
@@ -44,15 +44,16 @@ private ResourceParameters() {
       possibleParams.add("properties");
       possibleParams.add("clipGeometry");
     }
-    if (uri.contains("/contributions")) {
-      if (uri.contains("/count")) {
-        possibleParams.add("contributionType");
-      }
+    if (uri.contains("/contributions") && uri.contains("/count")) {
```

Comment:
hm, doesn't this now skip the `// removing deprecated params from newer endpoint` for the non-count contributions endpoints? AFAIK this step is still needed, isn't it?

Reply:
Right, I forgot they are also used in data extraction contributions endpoints.